### ISSUE TITLE
The linking pod no longer works with latest SDK

### ISSUE THREAD

**Issue by akash-rajput** [facebook/react-native-fbsdk#531]
## 🐛 Bug Report

The current version of fbsdk does not build with the latest Facebook SDK for iOS 5.0.0 The docs are misleading.

## To Reproduce

Following the steps mentioned in the readme

## Expected Behavior

Should work as expresed in the readme

**Comment by ollydixon** [comment]
Yep, this is critical. Our projects can no longer make updates to the App Store as they are rejected with 12.3

**Comment by ollydixon** [comment]
They should specify the exact version that works on the installation instructions.

**Comment by threadhack** [comment]
@akash-rajput have you solved this problem in latest Facebook SDK ?

**Comment by martin-badin** [comment]
any solution?

**Comment by zicrox** [comment]
I solved my case of iOS linking, I think it can be useful here: https://github.com/facebook/react-native-fbsdk/issues/528#issuecomment-505409226

**My Environment**
```
    IDE:
      Xcode: 10.1
    npmPackages:
      react: 16.8.6
      react-native: 0.59.9
      react-native-fbsdk: 0.10.1
    FacebookSDK: 5.1.1
```
